### ISSUE TITLE
Refraction and TravelTimeManager import file

### ISSUE THREAD

**Issue by stefanazzz** [gimli-org/gimli#278]
### Problem description

Import .sgt file works with pygimli 1.0.12 but not pygimli 1.1.1

### Your environment

Linux
Python 3.6
pyGIMLi 1.1.1+0.gbb685a8c (with local changes)
Installation w Conda package

### Steps to reproduce

Tell us how to reproduce this issue. Ideally, you could paste the code that produces the error:

```python
import matplotlib
%matplotlib notebook
import matplotlib.pyplot as plt
import pygimli as pg
from pygimli.physics import Refraction
# from pygimli.physics import TravelTimeManager
filename='./ShincliffeDyke.sgt'
ra = Refraction(filename)
print(ra)
ra.showData();
#####################
#Other option (same problem):
import matplotlib
%matplotlib notebook
import matplotlib.pyplot as plt
import pygimli as pg
# from pygimli.physics import Refraction
from pygimli.physics import TravelTimeManager
filename='./ShincliffeDyke.sgt'
ra = TravelTimeManager(filename)
print(ra)
ra.showData();

### Expected behavior
# with pygimli 1.0.12:

filename='./ShincliffeDyke.sgt'
ra = Refraction(filename)
print(ra)

Removed 11 values.
Data: Sensors: 33 data: 222
Refraction object
Data: Sensors: 33 data: 222
None

ra.showData();
# this plots arrival times and shot positions as in file sgt

### Actual behavior
# with pygimli 1.1.1:

ra = TravelTimeManager(filename)
print(ra)
<pygimli.physics.traveltime.TravelTimeManager.TravelTimeManager object at 0x7fa1eaa1bd50>

### Also  :
ra.showData(); 
02/01/21 - 13:23:33 - pyGIMLi - ERROR - None.drawVA(raplot.py:158)
zero traveltimes found.
#226 d attempts to plot a velocity instead of arrival time

###
ra = Refraction(filename)
print(ra)
<pygimli.physics.traveltime.TravelTimeManager.TravelTimeManager object at 0x7fa1ea94b450>

ra.showData(); 
02/01/21 - 13:40:05 - pyGIMLi - ERROR - None.drawVA(raplot.py:158)
zero traveltimes found.
### again attempts a velocity plot

> If possible, please add one or more labels to your issue, e.g. if you expect that your issue is rather a question than a problem with the code, please add the label "question".

```
Not sure question or bug or change in the way to implement?

Thanks!

![Screenshot at 2021-01-02 13-23-47](https://user-images.githubusercontent.com/45072850/103458592-91b81b00-4d01-11eb-906a-15a96d740c0d.png)
![Screenshot at 2021-01-02 13-23-12](https://user-images.githubusercontent.com/45072850/103458594-954ba200-4d01-11eb-8472-77a172465543.png)
![Screenshot at 2021-01-02 13-21-50](https://user-images.githubusercontent.com/45072850/103458597-9a105600-4d01-11eb-88cf-84662d554e12.png)

**Comment by halbmy** [comment]
Obviously there are zero travel times in the data file. Those data should be removed but I am not sure. Can you please attach the file so that we can check? Otherwise there is not much we can say.

**Comment by stefanazzz** [comment]
I tried with this file (attached) with no zeros. 
Same problem with pygimli 1.1.1+0.gbb685a8c (with local changes) but works with 1.0.12+0.g9962fe88 (with local changes).
I would need to use this with 1.1.1, which has updated meshing tools. 
Thanks for your support,

[SeisHoughall2016-Copy1.sgt.zip](https://github.com/gimli-org/gimli/files/5766367/SeisHoughall2016-Copy1.sgt.zip)

**Comment by halbmy** [comment]
These are the five leading lines:
```
33 # no. of geophones + shot points 
#x    y shots and geophs pos (geophs have decimal nonzero) 
-60.01	0 1 # shot 1 (item 1) - off end shot
0	0 # first geophone (item 2) 
0.01	0 # shot 2 - end shot (item 3)
```
Note that the first sensor (line 3) has three entries wherever the others have only two (the token line # x y should better match those as this is no comment but interpreted).

When I remove the extra ` 1`, the code
```
ra = TravelTimeManager(filename)
print(ra)
ra.showData()
```
works as it's supposed to be:
![image](https://user-images.githubusercontent.com/7136401/103573027-d0023580-4ecd-11eb-83e1-19e7e7fa1688.png)

**Comment by halbmy** [comment]
Nevertheless, I don't quite understand the double sensor at the zero position making the mesh unnecessarily fine. Try remove it. It also helps to sort sensors:
```
data = pg.DataContainer(filename, "s g") # marking s and g as indices
data.sortSensorsX()
```

**Comment by stefanazzz** [comment]
Hi, thanks for the correction on the datafile. But I expected to see this:
![image](https://user-images.githubusercontent.com/45072850/103642755-8bb57a80-4f4b-11eb-9fac-6d6326c064d6.png)

**Comment by stefanazzz** [comment]
the double sensor at 0 is same site used as source in one shot stack, and as geophone in another

**Comment by halbmy** [comment]
I've started years ago with the `Refraction` manager for refraction only, but as `TraveltimeManager` it became a bit more general, also for crosshole data, where the typical first break pick curves do not make sense. But you can show them by
```
ra.showData(firstPicks=True)
```
![image](https://user-images.githubusercontent.com/7136401/103672470-a3f1bd80-4f7c-11eb-8151-c1e8e1332ef1.png)

**Comment by halbmy** [comment]
> the double sensor at 0 is same site used as source in one shot stack, and as geophone in another

Well, you should better use only one sensor for both and consequently the same index. It might be a bit uncommon in traveltime tomography where typically shots and geophone positions are in separate files, but the format is originated from ERT where electrodes are naturally for both current and voltage. Maybe we should enable identical sensor positions in files and re-index when loading.

**Comment by stefanazzz** [comment]
Yes, I did not think of that but I can use the same index as shot in one gather and as geophone in another. Makes sense. 
The firstPicks=True works to display the arrival times, 
however i have now an issue with (also used to work in 1.0.12):

ra.createMesh(quality=24,paraMaxCellSize=6.0)

which gives the error:

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-36-e1fb8a80dcf9> in <module>
      1 filename='./SeisHoughall2016-Copy1.sgt'
      2 ra = TravelTimeManager(filename)
----> 3 ra.createMesh(quality=24,paraMaxCellSize=6.0)#,depth=50)
      4 ra.invert(zWeight=0.95,maxIter=5)#,depth=40.0)

~/anaconda3/envs/geo02/lib/python3.7/site-packages/pygimli/physics/traveltime/TravelTimeManager.py in createMesh(self, data, **kwargs)
    203             pg.critical('Please provide a data file for mesh generation')
    204 
--> 205         return pg.meshtools.createParaMesh(data.sensors(),
    206                                            boundary=0, **kwargs)
    207 

AttributeError: 'NoneType' object has no attribute 'sensors'

I don't understand why if the filename is passed to TravelTimeManager and "ra" is created, it now asks for a data file?
I also tried this: data = pg.DataContainer(filename, "s g t") but does not help

Thanks again and sorry for the long issue report...
